NEWS.md

```diff
@@ -22,6 +22,9 @@ See the [Shadowserver Reports API bot's documentation](https://intelmq.readthedo
 #### GitHub Collector
 GitHub removed the basic `Username/Password` Authentication in favor of personal access tokens. So the GitHub Collector uses an Personal Access Token for authentication [Github Documentation: Generate a personal access token](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token)
 
+#### URL Expert
+This is a completely new bot that extracts additional fields from `source.url` and/or `destination.url`. It can fill in IP, FQDN, Port, URL Path, Account and Application/Transport Protocol. This bot also replaces URL2FQDN Expert bot which is to be removed in IntelMQ version 4.0.0.
+
```

Comment:
We previously did not announce new bots here. The NEWS file is for sysadmins with notable changes in the software and instructions on upgrades

Reply:
Ok, removed.

Reply:
@aaronkaplan See this about NEWS. I am fine with either way, just let me know who should I listen to. :smile: